src/ingest/events/post.js

```diff
@@ -140,7 +142,7 @@ module.exports = async (req, res) => {
 				headers: req.headers,
 				error: err.stack || err,
 			})
-		);
-		return response.internalServerError(req, res, err);
+		)
+		return response.internalServerError(req, res, err)
 	}
-};
+}
```

Comment:
It seems that all semicolons have been removed. (https://github.com/swrlab/swr-prettier-config/blob/583af8ad205ee5b864b1c2fa63ca836cf817b73e/index.js#L5)